src/Serializers/TermMapSerializer.js

```diff
@@ -23,12 +23,12 @@ MODULE.TermMapSerializer = util.inherit( 'WbTermMapSerializer', PARENT, {
 	 * @throws {Error} if termMap is not a TermMap instance.
 	 */
 	serialize: function( termMap ) {
+		var termSerializer = require( './TermSerializer' );
```

Comment:
This still returns a constructor function, so this should read
```suggestion
		var TermSerializer = require( './TermSerializer' );
```
and go to the top of the file.